crowbar_framework/app/controllers/api/restart_management_controller.rb

```diff
@@ -37,7 +37,8 @@ def configuration
   api :POST, "/api/restart_management/restarts", "Clean the service restart flags"
   header "Accept", "application/vnd.crowbar.v2.0+json", required: true
   param :node, String, desc: "Node name", required: true
-  param :service, String, desc: "Service to clean restart flag for", required: true
+  param :cookbook, String, desc: "Cookbook to clean restart flags for", required: false
```

Comment:
This change may break backwards compatibility since you changed the API. Before it was iterating over all cookbooks, now you have to pass one. Are we ok to introduce this change now?

Reply:
I think it's fine as the feature is clearly marked as experimental